### PR TITLE
update django-product-details and set to https (bug 1113852)

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: lib/settings_base.py [hunk 1]
```diff
@@ -157,6 +157,7 @@ def lazy_langs(languages):
 
 # Where product details are stored see django-mozilla-product-details
 PROD_DETAILS_DIR = path('lib', 'product_json')
+PROD_DETAILS_URL = 'https://svn.mozilla.org/libs/product-details/json/'
```
Comment (magopian) [inline lib/settings_base.py]: I've seen in the bug there'll be https://product-details.mozilla.org/ soon (?), which'll be even better ;)

File: requirements/prod.txt [hunk 2]
```diff
@@ -102,7 +103,6 @@ urllib3==1.9
 -e git+https://github.com/yohanboniface/jingo-minify.git@amo.2014.09.03#egg=jingo_minify
 -e git+https://github.com/mozilla/django-moz-header.git@djangoapp#egg=django-moz-header
 -e git+https://github.com/washort/nuggets.git@02798dfce84030fca64775eaf56e92400f394e4f#egg=nuggets
--e git+https://github.com/fwenzel/django-mozilla-product-details@f5a3c3c846fb75e12ad890b22ed5375d5b85ccb4#egg=django-mozilla-product-details
```
Comment (magopian) [inline requirements/prod.txt]: Hurray! :dancers: one down

**Comment by magopian** [comment]
r+, thanks!

File: lib/settings_base.py (hunk 1 above)
Comment (pmclanahan) [inline lib/settings_base.py]: :+1: :cake:

**Pull request merged**